metrics/examples/prometheus/alerting-rules.yaml

```diff
@@ -7,6 +7,14 @@ data:
     groups:
     - name: kafka
       rules:
+      - alert: RunningOutOfSpace
+        expr: kubelet_volume_stats_available_bytes < 5368709120
```

Comment:
We need the test of whether the metric is available or not. Can we append conditions here @ppatierno ?